_schema.graphql

```diff
@@ -5660,6 +5669,7 @@ input EnableSecondFactorInput {
 type EnableSecondFactorPayload {
   # A unique identifier for the client performing the mutation.
   clientMutationId: String
+  recoveryCodes: [String!]
```

Comment:
@dblandin I wanted to double-check that this was OK to add this to the Metaphysics schema.